static/app/main.tsx

```diff
@@ -8,19 +8,24 @@ import {RouteContext} from 'sentry/views/routeContext';
 
 import {PersistedStoreProvider} from './stores/persistedStore';
 
+/**
+ * Renders our compatability RouteContext.Provider. This will go away with
+ * react-router v6.
+ */
+function renderRouter(props: any) {
```

Comment:
is there a type we can use here? 😄

Reply:
react router v3 types leave a lot to be desired 😬